Connectors/src/CosmosDb/scaffold/cloudfoundry.py

```diff
@@ -0,0 +1,16 @@
+from pysteel import cloudfoundry
+
+
+def setup(context):
+    """
+    :type context: behave.runner.Context
+    """
+    cf = cloudfoundry.CloudFoundry(context)
+    # remove previous app
+    app = 'cosmosdb-connector'
+    cf.delete_app(app)
+    # create service
+    service = 'csb-azure-cosmosdb-sql'
+    plan = 'small'
```

Comment:
Does this sample need the small plan, or can it use mini? I'd think since most of the samples are only dealing with a couple of records, we could use the smallest plan available

Reply:
I hadn't seen the "mini" plan. That one works too.